File: plugin/plugin-sys-webManager/src/main/java/com/dasoops/dasserver/plugin/webManager/entity/vo/LoginVo.java
package com.dasoops.dasserver.plugin.webManager.entity.vo;

import com.dasoops.common.entity.vo.base.BaseVo;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * @Title: LoginVo
 * @ClassPath com.dasoops.imageManagerServer.user.model.vo.LoginVo
 * @Author DasoopsNicole@Gmail.com
 * @Date 2022/12/26
 * @Version 1.0.0
 * @Description: 登录vo
 */
@EqualsAndHashCode(callSuper = true)
@Data
public class LoginVo extends BaseVo {

    private String token;

}

File: plugin/plugin-sys-webManager/src/main/java/com/dasoops/dasserver/plugin/webmanager/entity/vo/register/LoginVo.kt
package com.dasoops.dasserver.plugin.webmanager.entity.vo.register

import io.swagger.annotations.Api
import io.swagger.annotations.ApiModel

/**
 * @title LoginVo
 * @classPath com.dasoops.imageManagerServer.user.model.vo.LoginVo
 * @author DasoopsNicole@Gmail.com
 * @date 2022/12/26
 * @version 1.0.0
 * @description 登录vo
 */
@Api("Register")
@ApiModel("登录Vo")
data class LoginVo(
    /**
     * 令牌
     */
    var token: String,

    /**
     * 注册表id
     */
    var registerId: Long,

    /**
     * 名字
     */
    var name: String
)